doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -97,7 +97,28 @@ public virtual async Task<Response<MetricFeedback>> GetMetricFeedbackValuesAsync
 If the suggested rename does not apply to your scenario, you could pick the best name fitting your API.
 
 ## Implement the method
-You could call the protocol method inside your convenience method and map the returned raw response to model. See different scenarios and corresponding examples below. 
+You could call the protocol method inside your convenience method and map the returned raw response to model.
```

Comment:
Let's start this section with a line:

Inside a convenience method, you will do the following:
1. Optionally open a `DiagnosticScope`
2. Serialize any input model to `RequestContent`
3. Call the protocol method
4. Deserialize any response body to Response<T>

Reply:
Let's then break up the guidance that follows into sections that are numbered for each of these steps.